### ISSUE TITLE
Tic-Tac-Toe Smart Contract

### ISSUE THREAD

**Issue by Debopom-Banerjee** [Vikash-8090-Yadav/Solidity-Pathshala#182]
### Project title
Tic Tac Toe smart contract using solidity

### Project Description
The contract will contain functionalities like making a move one-by-one by two players and also will display the winner at the end. 

### Domain

- [x] Solidity
- [] Python
- [] JavaScript
- [] Golang
- [] Other, feel free to mention below

#### Are you contributing under any open-source program ?
Not right now except SWOC
<hr/>

**Comment by Vikash-8090-Yadav** [comment]
Great just tell me the folder name in which u want to add .

**Comment by Debopom-Banerjee** [comment]
Yeah I guess Level 2 can be a good fit for this? What's your opinion?

**Comment by Vikash-8090-Yadav** [comment]
Gr8  add this in Level2 .

**Comment by Debopom-Banerjee** [comment]
@DevSwayam  Hey I am already working on this issue. Please try some other one.
Thanks
